.circleci/config.yml

```diff
@@ -402,11 +402,11 @@ jobs:
           command: |
               export PATH=$HOME/miniconda/bin:$PATH
               . activate habitat;
-              if [ ! -d ./habitat-api ]
+              if [ ! -d ./habitat-lab ]
               then
-                git clone -q https://github.com/facebookresearch/habitat-api.git
+                git clone -q https://github.com/facebookresearch/habitat-lab.git
```

Comment:
nit: we should add --depth 1 so it runs faster (aka does a shallow clone)

Reply:
HabLab doesn't have deps submodules.

Reply:
That's now what it means by shallow. Shallow means it only checkouts the most recent files, not the history or older commits.